src/pax.nim

```diff
@@ -1,32 +1,39 @@
 import therapist
 import cli/clr, cli/prompt
-import cmd/add, cmd/expo, cmd/impo, cmd/init, cmd/list, cmd/remove, cmd/update, cmd/upgrade, cmd/version
+import cmd/add, cmd/expo, cmd/impo, cmd/init, cmd/list, cmd/remove, cmd/update,
+    cmd/upgrade, cmd/version
 
 let commonArgs = (
-  strategy: newStringArg(@["-s", "--strategy"], choices = @["recommended", "newest"], defaultVal = "recommended", help = "how pax determines the version to install"),
+  strategy: newStringArg(@["-s", "--strategy"], choices = @["recommended",
```

Comment:
did you format this with `nimpretty` or by hand? I haven't actually been using a formatter, maybe I should take a look so the output looks a bit better..

Reply:
I'm actually just using the Nim extension for vscode, the only configuration I have is as follows:
```json
    "editor.codeActionsOnSave": {
        "source.fixAll": true
    },
    "nim.lintOnSave": false
    ```
    However, it still seems to do some amount of cleanup, mostly just wrapping the lines after about 80 characters.